identity/client/src/pages/users/detail/authorization/List.tsx

```diff
@@ -47,23 +112,150 @@ const List: FC<AuthorizationsListProps> = ({ user, loadingUser }) => {
+                      cursor: "pointer",
+                      padding: "8px 16px",
+                      borderRadius: "4px",
+                      display: "flex",
+                      alignItems: "center",
+                    }}
+                  >
+                    <Add style={{ marginRight: "8px" }} />
+                    Add Authorizations
+                  </button>
+                </TableToolbarContent>
+              </TableToolbar>
+              <Table {...getTableProps()}>
+                <TableHead>
+                  <TableRow>
+                    <TableExpandHeader />
+                    {headers.map((header) => (
+                      <TableHeader {...getHeaderProps({ header })}>
+                        {header.header}
+                      </TableHeader>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {rows.map((row) => {
+                    return (
+                      <React.Fragment key={row.id}>
+                        <TableExpandRow
+                          {...getRowProps({ row })}
+                          onClick={() => handleExpand(row.id)}
+                          isExpanded={!!expandedRows[row.id]}
+                        >
+                          <TableCell>
+                            {
+                              row.cells.find(
+                                (cell) => cell.info.header === "resourceType",
+                              )?.value
+                            }
+                          </TableCell>
+                        </TableExpandRow>
+                        {expandedRows[row.id] && (
+                          <TableExpandedRow colSpan={headers.length + 2}>
+                            {/* Displaying list of permissions */}
+                            {dataMap[row.id]?.permissions && (
+                              <ul>
+                                {dataMap[row.id].permissions.map(
+                                  (permission, index) => (
+                                    <React.Fragment key={index}>
+                                      <li
+                                        style={{
+                                          display: "flex",
+                                          justifyContent: "space-between",
+                                          alignItems: "center",
+                                          padding: "10px 0",
+                                        }}
+                                      >
+                                        <div>
+                                          <strong>Permission Type:</strong>{" "}
+                                          {permission.permissionType}
+                                          <br />
+                                          <strong>Resource IDs:</strong>{" "}
+                                          {permission.resourceIds.join(", ")}
```

Comment:
Only style with CSS. [Carbon has specific typesets for such cases](https://carbondesignsystem.com/elements/typography/type-sets/)